coredb-operator/src/apis/coredb_types.rs

```diff
@@ -47,7 +47,10 @@ pub struct CoreDBSpec {
 #[allow(non_snake_case)]
 pub struct CoreDBStatus {
     pub running: bool,
-    pub extensions: Option<Vec<Extension>>,
+    #[serde(default = "defaults::default_extensions_updating")]
+    pub extensionsUpdating: bool,
+    #[serde(default = "defaults::default_extensions")]
+    pub extensions: Vec<Extension>,
```

Comment:
changing the default here to the empty array, rather than making them optional. this puts status in line with how it is defined in spec